barcode/barcode_api.py

```diff
@@ -0,0 +1,30 @@
+from uber.api import fields as attendee_fields
```

Comment:
Uh oh.  I totally get why you used these fields, but I hadn't intended ``fields`` to be used elsewhere, so I totally should have prefixed it with an underscore to make that clearer.  The only problem with having it imported like this is that now if I want to change or rename this field, it could break this code.

Not a problem yet, but definitely something I'll need to keep in mind in the future - I'll try to be better about prefixing things with underscores to advertise "importing or using this directly is dangerous since the author of this module might change it in the future without warning".